Add new sample with the implementation of the batch operations (#4)
southworks/Microsoft-Teams-Samples: samples/bot-batch-operations/csharp/README.md

You can interact with this bot in Teams by sending it a message, or selecting a command from the command list. The bot will respond to the following strings.

1. **Show Welcome**
  - **Result:** The bot will send the welcome card for you to interact with
  - **Valid Scopes:** personal, group chat, team chat
    
![image](Images/WelcomeCard.png)

2. **Message a List of Users**
- **Result:** The bot will send a message to each corresponding user from the list of user IDs provided in the dialog and return the operation ID.
- **Valid Scopes:** personal, group chat, team chat

![image](Images/MessageAListOfUsers.png)

3. **Send message to all users in a tenant**
- **Result:** The bot will send a 1-on-1 message to each user in the current tenant and return the operation ID
- **Valid Scopes:** personal, group chat, team chat

![image](Images/MessageAllUsersInTenant.png)

4. **Message All Users in a Team:**
- **Result:** The bot will send a 1-on-1 message to each user in the team corresponding to the ID provided in the dialog and return the operation ID
- **Valid Scopes:** personal, group chat, team chat

![image](Images/MessageAllUsersInTeam.png)

5. **Message A List of Channels**

> - **Result:** The bot will send a message to each corresponding channel from the list of channel IDs provided in the dialog (by default the current channel) abd return the operation ID.

Reviewer: ```suggestion
- **Result:** The bot will send a message to each channel from the list of channel IDs provided in the dialog (by default the current channel) and return the operation ID.
```

- **Valid Scopes:** personal, group chat, team chat

![image](Images/MessageAListOfChannels.png)

6. **Get Operation State**
- **Result:** The bot will open a dialog and ask for the operation ID and return if the corresponding operation is Ongoing, Completed, or Failed
- **Valid Scopes:** personal, group chat, team chat

![image](Images/GetOperationState.png)

7. **Get Failed Entries Paginated**

> - **Result:** The bot will show a list of the failed entries, and provide the continuation token to see multiple pages

Reviewer: ```suggestion
- **Result:** The bot will show a list of the failed entries, and provide the continuation token in case of multiple pages of results
```
